aspnetcore/security/authentication/certauth.md

```diff
@@ -14,6 +14,8 @@ uid: security/authentication/certauth
 
 [Configure your server](#configure-your-server-to-require-certificates) for certificate authentication, be it IIS, Kestrel, Azure Web Apps, or whatever else you're using.
 
+This document only covers IIS, Kestrel, and Azure Web Apps. NGINX, however [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/22054) contains some hints.
```

Comment:
```suggestion
This document only covers IIS, Kestrel, and Azure Web Apps. NGINX, however [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/22054) contains some hints on NGINX configuration.
```

Reply:
I'm not sure "some hints" is what we should be pointing at 🙃

We should find an nginx person internally and figure out how to do it well, I just don't know who that would be.